.github/workflows/ci.yml

```diff
@@ -11,7 +11,7 @@ on:
 
 jobs:
   build:
-    name: Python==${{ matrix.python-version }}
+    name: Python==${{ matrix.python-version }} | ${{ matrix.django-version }}
```

Comment:
@yashikajotwani12 why did you made this change? This module is not dependent on Django.

Reply:
Maybe I made it by mistake, I will take care next time, Thanks for the information